packages/react-router-dev/config/defaults/entry.server.node.tsx

```diff
@@ -45,8 +46,10 @@ export default function handleRequest(
           );
 
           pipe(body);
+          clearTimeout(timeoutId);
```

Comment:
Clearing here means `abort` will never be called in streaming scenarios that exceed the time limit since `onShellReady` fires when the shell is ready but there's still pending stuff to be streamed down, so this is basically clearing the time out at the start of the streamed content, regardless of how long it actually takes.